src/main/java/tohru/exception/TohruException.java

```diff
@@ -0,0 +1,16 @@
+package tohru.exception;
```

Comment:
TohruException could be split into a few further exceptions classes to specify what type of error? E.g TohruStringNotFound